contrib/autocompletion/README

```diff
@@ -0,0 +1,17 @@
+Bash and Zsh completion
```

Comment:
suggest add it to the document also.

Reply:
Which document?

Do you mean README or to the install docs for the website?

If we want to add it to the website install docs we're going to have to explain how to install auto complete scripts. This might get complex.